lib/fittings/swagger_authz.js

```diff
@@ -0,0 +1,67 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var injector = require('../../index.js').injector;
+var _ = injector.get('_');    // jshint ignore:line
+var Promise = injector.get('Promise');    // jshint ignore:line
+var debug = require('debug')('swagger:authz');
+var Error = injector.get('Errors'); // jshint ignore:line
+
+function loadSwagger(authz, swaggerDef) {
+    var promises = [];
+    var methods = ['get', 'put', 'post', 'delete', 'options', 'head', 'patch'];
+    _.forOwn(swaggerDef.paths, function(value, resourceName) {
+        _.forEach(methods, function(method) {
+            if(_.has(value, method) && _.has(value[method], 'x-privileges')) {
+                _.forEach(value[method]['x-privileges'], function(role) {
+                    var resource = swaggerDef.basePath + resourceName;
+                    debug('adding ACL: %s:%s:%s', role, resource, method);
```

Comment:
Ah gotcha - since we're adding something new in terms of how to get to this debugging detail, can you toss a relevant note in the repo's README or something? Somewhere to know how to get to these debugging statements and make them visible since we're using an alternate convention for the swagger pieces?